irohad/ametsuchi/impl/postgres_block_index.cpp

```diff
@@ -5,125 +5,128 @@
 
 #include "ametsuchi/impl/postgres_block_index.hpp"
 
-#include <unordered_set>
+#include <numeric>
 
-#include <boost/range/adaptor/filtered.hpp>
+#include <boost/format.hpp>
 #include <boost/range/adaptor/indexed.hpp>
-#include <boost/range/algorithm/for_each.hpp>
-#include <boost/range/numeric.hpp>
 #include "common/types.hpp"
```

Comment:
Please remove if it is unused.

Reply:
types.hpp is used for various types in function parameters

Reply:
`interfaces/common_objects/types.hpp` is used indeed, but I am not sure about `common/types.hpp`